ipn/localapi/localapi.go

```diff
@@ -791,6 +796,54 @@ func (h *Handler) serveUploadClientMetrics(w http.ResponseWriter, r *http.Reques
 	json.NewEncoder(w).Encode(struct{}{})
 }
 
+func (h *Handler) serveTkaStatus(w http.ResponseWriter, r *http.Request) {
```

Comment:
should this one be checking the method too? Perhaps localapi should be calling this as a `GET`?

Reply:
Done.